README.md

```diff
@@ -316,8 +320,54 @@ The Realm Dart package is `realm_dart`
 
     _*This file should be committed to source control_
 
-* For more usage of Realm Dart see the Realm Flutter usage above.
+* For more information on using the Realm Dart Standalone see the Realm Flutter usage above.
+
+# Sync data with Realm Flutter and Dart using Device Sync
+
+This section is about how to use the Realm with [Device Sync](https://www.mongodb.com/docs/realm/sdk/flutter/sync/) and how to connect to [Atlas App Services](https://www.mongodb.com/docs/realm/sdk/flutter/app-services/).
+
+## Set up Atlas App Services
+
+1. Create an account on [cloud.mongodb.com](https://cloud.mongodb.com) - follow the instructions: [Register a new Atlas Account](https://www.mongodb.com/docs/atlas/tutorial/create-atlas-account/#register-a-new-service-account).
+1. Create a new App following the instructions here: [Create an App with Atlas App Services UI](https://www.mongodb.com/docs/atlas/app-services/manage-apps/create/create-with-realm-ui).
+1. Read [Authentication Providers](https://www.mongodb.com/docs/atlas/app-services/authentication/providers/) to see how to configure the appropriate authentication provider type.
+1. Go to the **Device Sync** menu and [Enable Flexible Sync](https://www.mongodb.com/docs/atlas/app-services/sync/configure/enable-sync/#enable-flexible-sync).
+1. [Find and Copy the App ID](https://www.mongodb.com/docs/atlas/app-services/reference/find-your-project-or-app-id/) of your new application.
+
+## Use Device Sync with the Realm
+
+1. Initialize the App Services `App` client and authenticate a user.
+
+   ``` dart
+   String appId = "<Atlas App ID>";
+   final appConfig = AppConfiguration(appId);
+   final app = App(appConfig);
+   final user = await app.logIn(Credentials.anonymous());
+   ```
+1. Open a synced realm.
+   ``` dart
+   final config = Configuration.flexibleSync(user, [Task.schema]);
+   final realm = Realm(config);
+   ```
+
+1. Add a sync subscription and write data.
+
+   ``` dart
+   realm.subscriptions.update((mutableSubscriptions) {
+   mutableSubscriptions.add(realm.query<Task>(r'status == $0 AND progressMinutes == $1', ["completed", 100]));
+   });
+   await realm.subscriptions.waitForSynchronization();
+   realm.write(() {
+     realm.add(Task(ObjectId(), "Send an email", "completed", 4));
+     realm.add(Task(ObjectId(), "Create a meeting", "completed", 100));
+     realm.add(Task(ObjectId(), "Call the manager", "init", 2));
+   });
+   realm.close();
+   ```
+
+   Only data matching the query in the subscription will be synced to the server and only data matching the subscription will be downloaded to the local device realm file.
```

Comment:
```suggestion
   
   Only data matching the query in the subscription will be synced to the server and only data matching the subscription will be downloaded to the local device realm file.

   ``` dart
   realm.subscriptions.update((mutableSubscriptions) {
   mutableSubscriptions.add(realm.query<Task>(r'status == $0 AND progressMinutes == $1', ["completed", 100]));
   });
   await realm.subscriptions.waitForSynchronization();
   realm.write(() {
     realm.add(Task(ObjectId(), "Send an email", "completed", 4));
     realm.add(Task(ObjectId(), "Create a meeting", "completed", 100));
     realm.add(Task(ObjectId(), "Call the manager", "init", 2));
   });
   realm.close();
   ```
```

Reply:
moved description before code example